data/translations/es/state-data.json

```diff
@@ -25,7 +25,7 @@
     "default_ip_deadline": "hasta 29 días antes del día de las elecciones",
     "default_online_deadline": "hasta 29 días antes del día de las elecciones",
     "default_mail_postmarked_deadline": "debe tener el sello del correo 29 días antes del día de las elecciones",
-    "more_info_link_override": "https://azsos.gov/elections/voting-election/register-vote-or-update-your-current-voter-information-sp"
+    "more_info_link_override": "https://azsos.gov/elections/voters/registering-vote/registerupdate-vote"
```

Comment:
We don't need this override if it is not a translated link. It will use the default. Let's remove it.